app/src/main/res/drawable/ic_check_24dp.xml

```diff
@@ -2,6 +2,7 @@
   android:width="24dp"
   android:height="24dp"
   android:tint="#00645C"
+  android:autoMirrored="true"
```

Comment:
This should not be reversed. Only icons which represent direction should be reversed.

Reply:
Done.